src/components/LocalData.ts

```diff
@@ -13,6 +13,7 @@ export interface ILocalDataV1 {
     scores: IScores;
     settings: {
         selectedCases: number[];
+        selectedGenders: number[];
```

Comment:
I think this type is problematic if the user has already used the app without this new feature: this should potentially be undefined:

```suggestion
        selectedGenders: number[] | undefined;
```

And down in `loadLocalData` I'd make sure to load the default values for selectedGenders if it's undefined, otherwise if a user has used the app before, now they'd have no genders selected by default.